src/app/Shared/Model/Isolate.model.ts

```diff
@@ -165,11 +164,16 @@ export interface IsolateDTO {
                 value: string;
                 active: boolean;
             };
-        }; */
+        };
```

Comment:
Reject? Are these API changes?
The DTO should exactly reflect the data you get from the API.

Reply:
As it is now, it is the current api.